index.html

```diff
@@ -0,0 +1,130 @@
+  <!--  -->
+  <main>
+     <section id="home" class="first-sction">
+      <h3 class="hello">"Hello everyone"</h3>
+      <h1 class="ralph">RALPH03 IS STUDY PLATFORM <br>YOU CAN ACCESS ANYWHERE</h1>
+      <P class="first-p">Online education is the newly evolved form of education<br> where students study from their homes with the help <br> of computers and the internet. join us.</P>
+      <h3 class="date">2022.10.15(THUR) ~ (16) FRI</h3>
+        <P class="first-body">@National Museum of Korea, Art Center Nabi and More</P>
+    </section>
+    <!--  -->
+<section id="program" class="mes">
+  <div >
+    <div class="main-div">
+      <h3 class="main-program">
+        Main program
+      </h3>
+      <hr class="line1">
+    </div>
+    <div class="maindiv-top">
+    <div class ="main-pro">
+      <a class="mee" href="#"><img src="./images/program_icon_01.png"  width ="45" height="35px" alt="logo"></a>
+    <h4 class="pro-topic">
+      Lecture
+    </h4>
+    <p class="pro-msg">
+      listen to lecturals from various countries about softwear development. We will tach you how to master coding
+    </p>
+    </div>
+    <div class ="main-pro">
+      <a class="mee" href="#"><img src="./images/program_icon_02.png"  width ="45" height="35px" alt="logo"></a>
+    <h4 class="pro-topic">
+      Online
+    </h4>
+    <p class="pro-msg">
+      listen to lecturals from various countries about softwear development. We will tach you how to master coding
+    </p>
+    </div>
+    <div class ="main-pro">
+      <a class="mee" href="#"><img src="./images/program_icon_03.png"  width ="45" height="35px" alt="logo"></a>
+    <h4 class="pro-topic">
+      Forum
+    </h4>
+    <p class="pro-msg">
+      listen to lecturals from various countries about softwear development. We will tach you how to master coding
+    </p>
+    </div>
+    <div class ="main-pro boder">
+      <a class="mee" href="#"><img src="./images/program_icon_04.png"  width ="45" height="35px" alt="logo"></a>
+    <h4 class="pro-topic">
+      classe
+    </h4>
+    <p class="pro-msg">
+      listen to lecturals from various countries about softwear development. We will tach you how to master coding
+    </p>
+    </div>
+    <div class ="main-pro">
+      <a class="mee" href="#"><img src="./images/program_icon_05.png"  width ="45" height="35px" alt="logo"></a>
+    <h4 class="pro-topic">
+      Ignite.
+    </h4>
+    <p class="pro-msg">
+      listen to lecturals from various countries about softwear development. We will tach you how to master coding
+    </p>
+    </div>
+    <button class="mobile" >Join Ralph Online Study</button>
+  </div>
+  </div>
+  
+  <label class="desktop" >See the whole program</label>
+</section>
+<!--  -->
+<section class="speak-section">
+  <div class="feature-line">
+    <h3>Feature Speakers</h3>
+  <hr class ="fa-skp-line">
+  </div>
+  <div id="speakersGrid">
+  </div>
+</section>
+  </main>
+  <script src="./script.js"></script>
+</body>
+</html>
```

Comment:
- [ Optional  ] I recommend you to add "partner " and "footer" section for Home page Desktop Screen size just like [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

The Design Template of Home page  Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/192115752-3bf4e2b1-96a6-434d-b815-94c286bcecf4.PNG)

Your Homepage Look Likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/192115786-7d14015b-f5a7-46f1-9969-48c8ba928572.PNG)